decidim-generators/spec/lib/generators_spec.rb

```diff
@@ -17,7 +17,8 @@ module Decidim
         let(:test_version) { "0.27.0.dev" }
 
         it "returns the develop branch" do
-          expect(subject.edge_git_branch).to eq("develop")
+          # back to "develop" once https://github.com/decidim/decidim/pull/13978 is accepted
+          expect(subject.edge_git_branch).to eq("feature/new-collaborative_texts-module")
```

Comment:
I will apply this suggestion. Is created for visibility. 
```suggestion
          expect(subject.edge_git_branch).to eq("develop")
```